doc/content/devices/atecc608a/transfer.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Transfer to new Join Server"
+description: ""
+weight: 2
+distributions: ["Enterprise", "Cloud"]
+---
+
+ATECC608A/B secure elements are provisioned to The Things Join Server. This Join Server supports activation with all LoRaWAN Backend Interfaces compliant Network Servers. You don't need to be a user of {{% tts %}} to activate devices with a secure element via The Things Join Server.
+
+<!--more-->
+
+If you don't want to use The Things Join Server, you can transfer to a new Join Server. There are two options for doing so, which are both explained in this section.
```

Comment:
I don't get this. The first part is saying that you do not have to use The Things Stack as a network server to be using The Things Join Server, which is fine. But then you are saying that the user should follow the guide if he does not want to use The Things Join Server? Do you mean if he wants to use The Things Stack as a network server but does not want to use The Things Join Server?

Reply:
Clarified this a bit